vscode/src/jdkDownloader.ts

```diff
@@ -28,9 +27,28 @@ import { promisify } from 'util';
 
 let customView: vscode.WebviewPanel;
 let logger: vscode.OutputChannel;
+const ALGORITHM = 'sha256';
+
+export const getSha256Checksum = async (url: string) => {
+  return new Promise((resolve, reject) => {
+      https.get(`${url}.${ALGORITHM}`, res => {
+          if (res.statusCode !== 200) {
+            return `HTTP Error ${res.statusCode} - ${res.statusMessage}`;
```

Comment:
Should it throw an error here with the message string being returned here?
That way the rejection handler could appropriately notify the user.